includes/class-amp-theme-support.php

```diff
@@ -1872,6 +1893,10 @@ public static function start_output_buffering() {
 			newrelic_disable_autorum();
 		}
 
+		if ( isset( $_GET[ self::DISABLE_POST_PROCESSING_QUERY_VAR ] ) && AMP_Validation_Manager::has_cap() ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
```

Comment:
Instead of introducing these as top-level query parameters, how about putting them all under some `amp_flags` query var? This would make it easier to scrub them from the URL and it would reduce opportunities for collision. So So here it could be checking `isset( $_GET['amp_flags'][ self::DISABLE_POST_PROCESSING_QUERY_VAR ] )`.

Reply:
That's a really good idea.

Reply:
https://github.com/ampproject/amp-wp/pull/3448/commits/19363fc23939a38173ecfe0bd4d4baad3de2583a adds an [amp_flags query var](https://github.com/ampproject/amp-wp/pull/3448/commits/19363fc23939a38173ecfe0bd4d4baad3de2583a#diff-2585472f207548f626a43a7ff3cab922R103), to access these query vars like you mentioned.

For example:
`https://loc.test/your-post/?amp&amp_flags[reject_all_errors]`